_includes/portfolio-gallery.html

```diff
@@ -1,8 +1,7 @@
 <div class="gallery-container portfolio-gallery-container">
-  {% assign filtered_posts = site.posts | where: 'layout', 'portfolio' | where: 'is_archived', 'false' %} {% for
-  post in filtered_posts %}
-  {% include card.html thumbnail=post.thumbnail description=post.description url=post.url alt=post.alt areas=post.areas title=post.title %}
-  {% endfor %}
+  {% assign filtered_posts = site.posts | where: 'layout', 'portfolio' | where: 'is_archived', 'false' %} 
+  {% assign class="non-alternating" | append: include.class %}
+  {% include card-group.html cards=filtered_posts class=class %}
```

Comment:
would it work to pass `class=non_alternating` directly on line 4?

Reply:
Thanks for the suggestion! Yes, that works too.